src/wffile.c

```diff
@@ -131,9 +131,18 @@ DWORD MKDir(
    else
    {
       dwErr = GetLastError();
+
+      // Unfortunatley CreateDirectoryEx does not support developer mode, so we have to create symbolic ourselves
+      if (ERROR_PRIVILEGE_NOT_HELD == dwErr && (GetFileAttributes(pSrc) & FILE_ATTRIBUTE_REPARSE_POINT))
+      {
+         CreateSymbolicLink(pName, pSrc, SYMBOLIC_LINK_FLAG_DIRECTORY | SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE);

```

Comment:
...this seems to be linking the new object to the old object (the one being deleted in a move or the one being copied.)  It presumably wants to extract the reparse data from `pSrc`, find where `pSrc` is linked to, and link `pName` to the same destination as `pSrc`, not `pSrc` itself.

Reply:
Sry. My bad. It was too late :-)